README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Opis projektu
 
-DO UZUPEŁNIENIA
+Przykładowa strona sklepu zakodowana za pomocą darmowego szablonu w celu nauki kodowania stron i nauki pracy w zespole.
```

Comment:
Opisz szerzej, jakie sa mozliwosci

Do tego polecalbym ominac "w celu nauki kodowania..." opisz to tak jakbys chcial to sprzedac komus dalej jako boilerplate dla sklepow

Reply:
Mistrzem marketingu nigdy nie byłem ale opis rozszerzyłem.